apps/preview/next/pages/showcases/NavbarTop/NavbarTop.tsx

```diff
@@ -26,40 +38,92 @@ export default function TopNav() {
     },
   ];
 
+  const search = (event: React.FormEvent<HTMLFormElement>, searchValue: string) => {
+    event.preventDefault();
+    alert(`Successfully found 10 results for ${searchValue}`);
+  };
```

Comment:
You have access to `inputValue` in this scope, so you don't have to pass it as an argument.
```suggestion
  const search = (event: React.FormEvent<HTMLFormElement>) => {
    event.preventDefault();
    alert(`Successfully found 10 results for ${inputValue}`);
  };
```

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2638/commits/eae1bae35fa1ae13223452be0af6064f452d573a